pkg/operator2/configmap.go

```diff
@@ -37,6 +38,20 @@ const stubMetadata = `
 }
 `
 
+func getMetadataStruct(route *routev1.Route) map[string]interface{} {
+	var ret map[string]interface{}
+
+	metadataJSON := getMetadata(route)
+	err := json.Unmarshal([]byte(metadataJSON), &ret)
+	if err != nil {
+		panic(err)
```

Comment:
comment stating this panic should never happen unless our static metadata is broken